src/libcsg/modules/io/growriter.cc

```diff
@@ -60,7 +60,7 @@ void GROWriter::Write(Topology *conf) {
     string resname =
         top->getBead(i)
             ->getResidueName();  // top->getResidue(resnr)->getName();
-    string atomname = top->getBead(i)->getName();
+    string atomname = top->getBead(i)->getType();
```

Comment:
what does this change do?

Reply:
I have gotten rid of all the setName and getName methods because they are ambiguous with the ids and the types of each object. They were simply being used like this 
Atom Type Carbon Id 1 

Name was being assigned something like this:
"Carbon#1" 
Which is redundant with the id and does not add anything useful.